### PR TITLE
ref(css): refactoring table styles

### PR DESCRIPTION
**Pull request by robinrendle**
Removing junk table styles: `.table-striped`, `.table-bordered`, `.table-responsive`

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationGroupDetails/groupTagValues.tsx [hunk 1]
```diff
@@ -149,7 +149,7 @@ class GroupTagValues extends AsyncComponent<
             />
           </HeaderButtons>
         </Header>
-        <table className="table table-striped">
```
Comment (robinrendle) [inline src/sentry/static/sentry/app/views/organizationGroupDetails/groupTagValues.tsx:152]: This is the only place that uses this class so I’m moving those styles into this component

File: src/sentry/static/sentry/less/includes/bootstrap/tables.less [hunk 2]
```diff
@@ -100,40 +100,6 @@ th {
...
-      }
-    }
-  }
-  > thead > tr {
-    > th,
-    > td {
-      border-bottom-width: 2px;
-    }
-  }
-}
```
Comment (robinrendle) [inline src/sentry/static/sentry/less/includes/bootstrap/tables.less:125]: Not using this class anywhere

File: src/sentry/static/sentry/less/includes/bootstrap/tables.less [hunk 3]
```diff
@@ -100,40 +100,6 @@ th {
...
-// Zebra-striping
-//
-// Default zebra-stripe styles (alternating gray and transparent backgrounds)
-
-.table-striped {
-  > tbody > tr:nth-of-type(odd) {
-    background-color: @table-bg-accent;
-  }
-}
-
```
Comment (robinrendle) [inline src/sentry/static/sentry/less/includes/bootstrap/tables.less:136]: Or this one

File: src/sentry/static/sentry/less/includes/bootstrap/variables.less [hunk 4]
```diff
@@ -121,8 +121,6 @@
 
 //** Default background color used for all tables.
 @table-bg: transparent;
-//** Background color used for `.table-striped`.
-@table-bg-accent: #f9f9f9;
 //** Background color used for `.table-hover`.
 @table-bg-hover: #f5f5f5;
 @table-bg-active: @table-bg-hover;
```
Comment (robinrendle) [inline src/sentry/static/sentry/less/includes/bootstrap/variables.less:126]: Gotta delete all these variables, but going to do them one by one

**Review by priscilawebdev — APPROVED** [review]
lgtm

**Comment by priscilawebdev** [comment]
@robinrendle Please have you checked `getsentry` as well? I saw that we are using "table-bordered " in one of our templates

**Pull request merged**